build/NuSpecs/Microsoft.WindowsAppSDK.Bootstrap.targets

```diff
@@ -8,7 +8,7 @@
   </PropertyGroup>
 
   <Target Name="BinPlaceBootstrapDll" Condition="'$(AppxPackage)' != 'true'" AfterTargets="Build">
-    <Copy SourceFiles="$(MSBuildThisFileDirectory)..\runtimes\lib\native\$(_WindowsAppSDKFoundationPlatform)\Microsoft.WindowsAppSDK.Bootstrap.dll" DestinationFolder="$(OutDir)" OverwriteReadOnlyFiles="true" ContinueOnError="false"/>
+    <Copy SourceFiles="$(MSBuildThisFileDirectory)..\runtimes\win-$(_WindowsAppSDKFoundationPlatform)\native\Microsoft.WindowsAppSDK.Bootstrap.dll" DestinationFolder="$(OutDir)" OverwriteReadOnlyFiles="true" ContinueOnError="false"/>
```

Comment:
>_WindowsAppSDKFoundationPlatform [](http://example.com/codeflow?start=68&length=32)

@DrusTheAxe  I've mentioned in your merged pull request. does it still break anycpu build?